koku/masu/test/celery/test_tasks.py

```diff
@@ -13,12 +16,25 @@
 from api.dataexport.models import DataExportRequest as APIExportRequest
 from api.dataexport.syncer import SyncedFileInColdStorageError
 from masu.celery import tasks
+from masu.celery.export import TableExportSetting
+from masu.database.reporting_common_db_accessor import ReportingCommonDBAccessor
 from masu.test import MasuTestCase
+from masu.test.database.helpers import ReportObjectCreator
 
 fake = faker.Faker()
 DummyS3Object = namedtuple('DummyS3Object', 'key')
 
 
+def dictify_table_export_settings(table_export_settings):
```

Comment:
Maybe move this helper function into one of the main code packages and call it from `upload_normalized_data` too, and then it can also get testing coverage.